### PR TITLE
Add pull request and bug report templates

### PR DESCRIPTION
**Pull request by shadeMe**

### CODE REVIEW

**Comment by shadeMe** [comment]
The Netlify errors are spurious as the new Markdown files are only pertinent to our GitHub workflow. So, we can ignore those failures.

File: .github/ISSUE_TEMPLATE/01_bugs.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: "\U0001F6A8 Submit a Bug Report"
+about: Use this template if you came across a bug or unexpected behaviour differing from the docs.
+---
+
+<!-- NOTE: For questions or install related issues, please open a Discussion instead. -->
```
Comment (svlandeg) [inline .github/ISSUE_TEMPLATE/01_bugs.md:6]: It should be made clear that they need to open a Discussion thread on spaCy though, right?
I see that we actually do have a "Discussions" tab, but were we planning on actually using that?

Comment (shadeMe) [reply .github/ISSUE_TEMPLATE/01_bugs.md:6]: Ah, good point. Honestly, I'm not sure what would be best going forward. Using the spaCy Discussions to handle Thinc-specific issues seems like overkill at the moment - We only get infrequent issues in this repo.

We could just remove that comment for the time being and reassess once/if we start seeing more traffic.

Comment (svlandeg) [reply .github/ISSUE_TEMPLATE/01_bugs.md:6]: That's fine for me as well, like you said the volume is not that high on Thinc.

**Review by svlandeg — APPROVED** [review]
Thanks!

**Pull request merged**